src/shared/metadata/cgroup_path_resolver.cc

```diff
@@ -47,15 +57,11 @@ StatusOr<std::string> CGroupBasePath(std::string_view sysfs_path) {
     }
   }
 
-  std::string cgv2_base_path = absl::StrCat(sysfs_path, "/cgroup");
-  struct statfs info;
-  auto fs_status = statfs(cgv2_base_path.c_str(), &info);
-  bool cgroupv2 = (fs_status == 0) && (info.f_type == CGROUP2_SUPER_MAGIC);
-
+  // TODO(ddelnano): this aids in testing the LegacyCGroupPathResolver. Determine what
+  // to do with this.
```

Comment:
I was worried about the possibility of `FLAGS_force_cgroup2_mode`'s default true value meant this code path could be used in production clusters. In theory, the caller should not parse a `cgv2_base_path` correctly (since the filesystem check didn't indicate cgroup v2). However, since both v1 and v2 can be active and this OVH issue shows that these cases are complicated, I'm a little worried about removing this.